beekeeper-path-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/model/EventModel.java

```diff
@@ -17,21 +17,7 @@
 
 import com.expediagroup.beekeeper.core.model.LifecycleEventType;
 
-public class EventModel {
+public interface EventModel {
```

Comment:
I think this can problably go completely, see https://github.com/ExpediaGroup/beekeeper/pull/51/files#r444041998